Sources/PCCurrentUser.swift

```diff
@@ -1147,3 +1148,27 @@ public enum PCRoomMessageFetchDirection: String {
 public typealias PCErrorCompletionHandler = (Error?) -> Void
 public typealias PCRoomCompletionHandler = (PCRoom?, Error?) -> Void
 public typealias PCRoomsCompletionHandler = ([PCRoom]?, Error?) -> Void
+
+//MARK: Beams
+
+private let pushNotifications: PushNotifications = PushNotifications()
+
+extension PCCurrentUser {
+    /**
+     Start PushNotifications service.
+     */
+    public func enablePushNotifications() {
+        let tokenProvider = BeamsTokenProvider(authURL: "https://beams-test-server.herokuapp.com/auth") { () -> AuthData in
+            return AuthData(headers: [:], urlParams: [:])
+        }
+
+        pushNotifications.start(instanceId: self.instance.id, beamsTokenProvider: tokenProvider)
+
+        try? pushNotifications.setUserId(self.id, completion: { (error) in
+            guard error == nil else {
+                print(error.debugDescription)
```

Comment:
This will be removed.
Should I use `PCLogger` here?

Reply:
Yep I'd use the logger infrastructure that already exists in the SDK if possible